mobile-widgets/qmlinterface.h

```diff
@@ -102,12 +111,36 @@ class QMLInterface : public QObject {
 
 public:
 	CLOUD_STATUS cloud_verification_status() { return (CLOUD_STATUS)prefs.cloud_verification_status; }
+	DURATION duration_units() { return (DURATION)prefs.units.duration_units; }
+	LENGTH length() { return (LENGTH)prefs.units.length; }
+	PRESSURE pressure() { return (PRESSURE)prefs.units.pressure; }
+	TEMPERATURE temperature() { return (TEMPERATURE)prefs.units.temperature; }
+	UNIT_SYSTEM unit_system() { return (UNIT_SYSTEM)prefs.unit_system; }
+	TIME vertical_speed_time() { return (TIME)prefs.units.vertical_speed_time; }
+	VOLUME volume() { return (VOLUME)prefs.units.volume; }
+	WEIGHT weight() { return (WEIGHT)prefs.units.weight; }
 
 public slots:
 	void set_cloud_verification_status(CLOUD_STATUS value) {  qPrefCloudStorage::set_cloud_verification_status(value); }
+	void set_duration_units(DURATION value) { qPrefUnits::set_duration_units((units::DURATION)value); }
+	void set_length(LENGTH value) { qPrefUnits::set_length((units::LENGTH)value); }
+	void set_pressure(PRESSURE value) { qPrefUnits::set_pressure((units::PRESSURE)value); }
+	void set_temperature(TEMPERATURE value) { qPrefUnits::set_temperature((units::TEMPERATURE)value); }
+	void set_unit_system(UNIT_SYSTEM value) { qPrefUnits::set_unit_system((unit_system_values)value); }
+	void set_vertical_speed_time(TIME value) { qPrefUnits::set_vertical_speed_time((units::TIME)value); }
+	void set_volume(VOLUME value) { qPrefUnits::set_volume((units::VOLUME)value); }
+	void set_weight(WEIGHT value) { qPrefUnits::set_weight((units::WEIGHT)value); }
```

Comment:
yeah, so here we see this in use... and I guess it's not as bad as it's used as `...units...set_weight()` so the `units` part may have felt repetitive. 
So the only one that might eventually deserve being renamed is `duration_units` into something like `duration_always_hours`